src/routes/mod/[modId]/new-version/+page.svelte

```diff
@@ -83,14 +84,24 @@
   {/if}
 </svelte:head>
 
-<h1 class="text-4xl my-4 font-bold">
-  New Version for
-  {#if $mod.fetching}
-    ...
-  {:else if !$mod.error}
-    {$mod.data.mod.name}
-  {/if}
-</h1>
+<div class="flex flex-wrap h-auto justify-between items-center">
+  <h1 class="text-4xl my-4 font-bold">
+    New Version for
+    {#if $mod.fetching}
+      ...
+    {:else if !$mod.error}
+      {$mod.data.mod.name}
+    {/if}
+  </h1>
+  <div>
+    <button
+      class="btn variant-ghost-primary"
+      title="View the description page for this mod"
+      on:click={() => goto(base + '/mod/' + modId)}>
+      Return to Description Page (discard new version)
+    </button>
```

Comment:
This probably should instead say "Back to Mod" and when clicked it confirms via popup modal.